frontend/src/styles/base.css

```
@@ -0,0 +1,109 @@
+  color: var(--ink);
+  background: var(--paper);
+  letter-spacing: -0.006em;
+  /* Korean line-breaking: break at eojeol (spaces), never mid-word.
+     overflow-wrap:break-word only rescues pathologically long tokens and,
+     unlike 'anywhere', does not collapse min-content to a single glyph. */
+  word-break: keep-all;
+  overflow-wrap: break-word;
+  text-wrap: pretty;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)